VoodooI2C/VoodooI2C/VoodooI2CDevice/VoodooI2CDeviceNub.cpp

```diff
@@ -201,6 +193,17 @@ IOReturn VoodooI2CDeviceNub::getDeviceResources() {
         return kIOReturnNotFound;
     }
 
+    IOPCIDevice *pci_device { nullptr };
+    bool force_polling { false };
+
+    pci_device = controller->nub->controller->physical_device.pci_device;
+
+    if (checkKernelArg("-vi2c-force-polling")) {
```

Comment:
This if statement can be refactored into:

```
bool force_polling  = (pci_device != nullptr) && checkKernelArg("-vi2c-force-polling");
```

Reply:
But then we have to check for `force-polling` property too.

Reply:
Right, missed the else, we want something like:

```
bool force_polling  =  checkKernelArg("-vi2c-force-polling") || ((pci_device != nullptr) && (pci_device->getProperty("force-polling") != nullptr));

```

Reply:
Got it, it's for readability. I'll refactor it. Thanks.